compiler/src/dotty/tools/dotc/ast/Desugar.scala

```diff
@@ -890,6 +890,9 @@ object desugar {
     }
   }
 
+  def enumGetters(getters: EnumGetters)(using Context): Tree =
+   flatTree(DesugarEnums.enumBaseMeths).withSpan(getters.span)
```

Comment:
That one would be generated directly by desugar.

Reply:
this is no longer part of the implementation